tests/D2L.CodeStyle.TestAnalyzers.Tests/D2L.CodeStyle.TestAnalyzers.Tests.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="NUnit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
+    <Compile Include="ServiceLocator\CustomTestServiceLocatorAnalyzerTests.cs" />

```

Comment:
This also seems weird. I probably shouldn't have tried to resolve merge conflicts normally on something this old...